modules/roles_profiles/manifests/profiles/gecko_t_linux_talos_generic_worker.pp

```diff
@@ -20,11 +20,9 @@
             }
 
             # TODO: move these lines to linux-base?
-            require linux_packages::py2
```

Comment:
These still need to be installed on the 1804 configuration.

Reply:
Fixed, 2204 config has its own file now.